packages/kotti-ui/source/kotti-popover/KtPopoverIconTextItem.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div
+		class="kt-popover-item"
+		:class="{
+			'kt-popover-item--is-disabled': isDisabled,
+		}"
+		v-on="$listeners"
+	>
+		<i v-show="icon" class="yoco" v-text="icon" />
+		<div>{{ label }}</div>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'KtPopoverIconTextItem',
+	props: {
+		icon: {
+			type: String,
+			default: null,
+		},
+		isDisabled: {
+			type: Boolean,
+		},
+		label: {
+			type: String,
+		},
+	},
+}
+</script>
+
+<style lang="scss">
```

Comment:
`scoped` please

Reply:
And for future reference, unless you have a **very strong reason**, please **always** `scope`